tests/core/full_node/test_mempool.py

```diff
@@ -377,6 +381,20 @@ async def test_invalid_block_index(self, two_nodes):
         assert status == MempoolInclusionStatus.PENDING
         assert err == Err.ASSERT_HEIGHT_ABSOLUTE_FAILED
 
+    @pytest.mark.asyncio
+    async def test_block_index_missing_arg(self, two_nodes):
```

Comment:
the `_missing_arg` tests ensure we correctly fail when conditions are missing argument(s)